examples/weaviate/personalized_description_with_weaviate_and_gemini_api.ipynb

```diff
@@ -0,0 +1,1119 @@
+{
```

Comment:
<p>Not sure what happened but when I ran this version of the notebook, <code>import weaviate</code> failed with a protobuf error. Might be a colab update? Or something in a library in the middle somewhere?</p><p> </p><p>If you can find a real fix, then that'd be great. If not, I worked around it by adding <code>pip install 'protobuf&gt;=5'</code> at the end of this pip install block.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Hey Mark, </p><p>Adding <code>pip install 'protobuf&gt;=5'</code> at the end didn't work for me. One workaround I found, was to upgrade Weaviate to <code>4.7.1</code> and install <code>python3.11</code>. Colab is currently on <code>Python 3.10.12</code> and I'm assuming that is the problem. </p><p> </p><p>It's not the prettiest workaround but I've added <code>!sudo apt-get install python3.11</code> to the beginning.  </p>